README.md

```diff
@@ -91,38 +91,77 @@ The front end of the integration is now complete.
-                    // will make the call.
-                    return false
-                })
-                .setBackendCompletion({ (data, error) in
-                    // when the callback has been invoked from the SDK, you can get the
-                    // data/error here. This needs to be defined even if it is empty.
-                })
-                .create()
-            
-        } catch {
-            // error management here
-            print(error)
-        }
+             guard let url = URL(string: "YOUR_CALLBACK_URL") else {
+                 return false
+             }
+             //Here we need to add as many scenarios as we want. each scenario is linked to a button in the Main.storyboard.
+             let scenario_1 = try ScenarioBuilder().setUseCaseID("yoti_btn_1")
+                 .setClientSDKID("YOUR_CLIENT_SDK_ID")
+                 .setScenarioID("YOUR_SCENARIO_ID_1")
+                 .setCallbackBackendURL(url)
+                 .create()
+             YotiSDK.add(scenario: scenario_1)
+             
+             let scenario_2 = try ScenarioBuilder().setUseCaseID("yoti_btn_2")
+                 .setClientSDKID("YOUR_CLIENT_SDK_ID")
+                 .setScenarioID("YOUR_SCENARIO_ID_2")
+                 .setCallbackBackendURL(url)
+                 .create()
+             YotiSDK.add(scenario: scenario_2)
+             
+         } catch {
+             // handle error code here
+         }
+    }
+```
+
+then in your viewController class call the static function ```public static func startScenario(for useCaseID: String, with delegate: YotiSDKDelegate) throws``` by passing it the useCaseID and self as delegate.
+
+your ViewController class should comply to YotiSDKDelegate and to BackendDelegate in order to get the callbacks when the app has opened or not.
+
+```extension ViewController: YotiSDKDelegate {
+    func yotiSDKDidFail(for useCaseID: String, with error: Error) {
+    	//handle here the error opening of Yoti app
+    }
+
+    func yotiSDKDidSucceed(for useCaseID: String, baseURL: URL?, token: String?, url: URL?) {
+        //Handle here the success of the opening of Yoti app for example by requesting a profile from the backend like below
+        //Get the specific scenario by calling  
+        let scenario = YotiSDK.scenario(for: useCaseID)
+        //request the backend to get the profile linked to a specific scenario by passing the token returned and self as delegate for a call back
+        YotiSDK.callbackBackend(scenario: scenario!, token: token!, with: self)
+    }
+
+    func yotiSDKDidOpenYotiApp() {
+    //Handle specific behaviour if needed when the Yoti App didOpen
+
+    }
+}
+```
+when the callback returns from the backend we get the data linked to the profile or the error in `func backendDidFinish(with data: Data?, error: Error?)`
+the data can be checked as JSON and parsed in order to display the profile.
```

Comment:
This is not part of the SDK, and depends on clients implementation

Reply:
@CharlesVu : Do you mean , we should not include this part inside this documentation?